lib/AST/Builtins.cpp

```diff
@@ -1027,7 +1027,8 @@ static ValueDecl *getAutoDiffApplyAssociatedFunction(
   AutoDiffParameterIndicesBuilder autoDiffBuilder(origFnTy);
```

Comment:
And 1027-1029 should be removed since they are no longer used.

Reply:
Oops, done in https://github.com/apple/swift/pull/25324/commits/866c6abe948bb44fb27c0d66957d2184fab16249.